### PR TITLE
new exercuses for Quest09, and fixes

### PR DESCRIPTION
**Pull request by LEEDASILVA**

### CODE REVIEW

File: doubtful/src/main.rs [hunk 1]
```diff
@@ -1,8 +1,14 @@
-// Write a function called `doubtful` that adds to every string passed
-// to it a question mark (?)
+/*
+## error types
```
Comment (MSilva95) [inline doubtful/src/main.rs:2]: change to mutability

File: panic/src/lib.rs [hunk 2]
```diff
@@ -1,10 +1,14 @@
-// Write a function call `open_file` that tries to open a file and panics if the file
-// doesn't exist
-use std::fs::{self, File};
+/*
+## error types
```
Comment (MSilva95) [inline panic/src/lib.rs]: change to panic

File: unwrap_and_expect/src/lib.rs [hunk 3]
```diff
@@ -18,26 +18,9 @@ Create the following functions that receives a vector :
   - `unwrap_or_else` that in case of error returns a the
```
Comment (MSilva95) [inline unwrap_and_expect/src/lib.rs:18]: change name to unwrap_and_expect

File: doubtful/src/main.rs (hunk 1 above)
Comment (MSilva95) [reply doubtful/src/main.rs:2]: change to doubtful

**Review by augusto-mantilla — APPROVED** [review]

**Pull request merged**